fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/RegexpExtractAll.java

```diff
@@ -36,12 +35,13 @@
  * ScalarFunction 'regexp_extract_all'. This class is generated by GenerateFunction.
  */
 public class RegexpExtractAll extends ScalarFunction
-        implements BinaryExpression, ExplicitlyCastableSignature, AlwaysNullable, PropagateNullLiteral {
+        implements ExplicitlyCastableSignature, PropagateNullable {
 
     public static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
-            FunctionSignature.ret(VarcharType.SYSTEM_DEFAULT)
+            FunctionSignature.ret(ArrayType.of(VarcharType.SYSTEM_DEFAULT, false))
```

Comment:
be's array's contains null should always be true

Reply:
Yes.